libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1485,7 +1485,7 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 		solAssert(!functionType->gasSet(), "");
 		solAssert(!functionType->bound(), "");
 
-		static map<FunctionType::Kind, std::tuple<u160, size_t>> precompiles = {
+		static map<FunctionType::Kind, std::tuple<unsigned, size_t>> precompiles = {
```

Comment:
This piece of code doesn't actually need h160/u160 or anything like that. The only use of this variable is `toString(address)` below for whiskers.